pilz_control/include/pilz_control/pilz_joint_trajectory_controller_impl.h

```diff
@@ -174,6 +174,15 @@ void PilzJointTrajectoryController<SegmentImpl, HardwareInterface>::triggerMovem
   JointTrajectoryController::setHoldPosition(this->time_data_.readFromRT()->uptime);
 }
 
+template <class SegmentImpl, class HardwareInterface>
+void  PilzJointTrajectoryController<SegmentImpl, HardwareInterface>::trajectoryCommandCB(const JointTrajectoryConstPtr& /*msg*/)
```

Comment:
The parameter name is commented out. Is this intended?

Reply:
See https://stackoverflow.com/questions/9175257/why-comment-parameter-names-rather-than-leave-it-as-it-is